website/static/js/projectSettings.js

```diff
@@ -59,6 +79,15 @@ var NodeCategorySettings = oop.extend(
                 err: error
             });
         },
+        updateTitleError: function() {
+           $('#titleInputMessage').html('Title cannot be blank.');
```

Comment:
Prefer to use observables bound to HTML elements rather than using jquery to manually manipulate the DOM.